src/fetch/payouts.py

```diff
@@ -153,6 +160,21 @@ def as_payouts(self) -> list[Transfer]:
             else 0
         )
 
+        result = []
+        if quote_reward_cow > 0:
+            try:
+                result.append(
+                    Transfer(
+                        token=Token(COW_TOKEN_ADDRESS),
+                        recipient=self.reward_target,
+                        amount_wei=quote_reward_cow,
+                    )
+                )
+            except AssertionError:
+                logging.warning(
+                    f"Invalid COW Transfer {self.solver} with amount={quote_reward_cow}"
+                )
```

Comment:
I don't think this is necessary here. If I remember correctly, we have it in other places because the `amount_wei` could be non-positive.

Reply:
Yes, they are not necessary. I will remove them.

Reply:
Should all the "try-catch" blocks be turned into if statements?
Only one of the 5 try statements can be removed without replacement.